### DISCUSSION TITLE
Fixed column while scrollable horizontally [v8]

### DISCUSSION THREAD

**Discussion by dufia** [TanStack/table#4033] [Q&A]
How would I go about implementing:
- first column is fixed
- the rest of the columns are scrollable horizontally if the container isn't wide enough

Does v8 support it or do I need to build my own implementation?

Cheers!

**Comment by colindjk** [comment]
So there's pinned columns, but afaik that really only helps with the state management aspect of pinning columns.

I'm currently working on getting a solution that integrates pinned columns logic in RT with a generic solution to keep an arbitrary # of columns "pinned" (or "frozen" for excel nomenclature). Could have some updates there if I figure something out :slightly_smiling_face:
